### DISCUSSION TITLE
Solana Jupiter Bot - yarn wizard cannot select Tokens

### DISCUSSION THREAD

**Discussion by searcher-py** [ARBProtocol/solana-jupiter-bot#22] [Q&A]
Hello Pavel how are you?

I corrected the Node.js version to 16 and could execute yarn start or wizard without anymore errors.

But when I reach in the UI from the wizard the selection of the Tokens I cannot select any.
The UI does not show me any Tokens that I may select neither in mainnet-beta nor in devnet
Have NOT the blue cursor (arrow) visible for selection
Have to break with Crtl-C

I am using Ubuntu Server 22.04 

Thank and greetings
Klaus
![Screen Shot 2022-11-03 at 12 58 46 PM](https://user-images.githubusercontent.com/117361355/199807158-33b1060b-c26e-42e2-af86-1c7bbe8caefe.jpg)
![Screen Shot 2022-11-03 at 12 50 52 PM](https://user-images.githubusercontent.com/117361355/199807171-b8947bfb-61b4-42d2-95c8-efef98376cd1.jpg)

**Comment by TatyOko28** [comment]
The issue likely stems from either:

Missing Token Data from the connected Solana network (mainnet-beta or devnet).
An incomplete setup of your bot or its dependencies.
A UI rendering problem (e.g., missing cursor or selection highlight).
Fix: Ensure the bot fetches tokens properly by verifying the RPC connection and checking for token list availability.
